docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-product-management-feature-walkthrough.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Marketplace Merchant Portal Product Management feature walkthrough
+description: This article provides reference information about product in the Merchant Portal.
+template: concept-topic-template
+---
+
+The *Marketplace Merchant Portal Product Management* feature allows merchants to manage product in the Merchant portal.
```

Comment:
Please add mode detailed information about functionality. (What exactly we can manage from the product data?)